bot.py

```diff
@@ -394,7 +394,7 @@ def get_aliases(dict, exclude=''):
 # deposit (split into 3 for easy copypasting address on mobile)
 DEPOSIT_TEXT="Your wallet address is:"
 DEPOSIT_TEXT_2="{0}"
-DEPOSIT_TEXT_3="QR: {0}"
+DEPOSIT_TEXT_3=":Account history and QR code: https://creeper.banano.cc/explorer/account/{0}"
 
```

Comment:
I sort of opt for keeping the QR code on the deposit screen, but for these strings the parameters are set in the actual command see deposit function line 819.

You could do something like this:

DEPOSIT_TEXT_3="QR: {0}\nExplorer: {1}" and then set the second parameter in the deposit command , or replace the QR code all together by removing the get_qr_url function

Reply:
I felt like having the QR on the explorer is enough. Loading the QR from an external Image always felt a little clunky, and basically no one is actually using this right now I guess. But yeah, we could keep it since mobile wallets will arrive soon. The Explorer can also add other convenience stuff like a link for direct vault payment  https://vault.banano.co.in/send?to=ban_1.. in case someone does not have registered the ban: prefix system-wide.  

You're correct about changing the get_qr_url thing. Will look into this